examples/units_and_coordinates/PSP_trag_sunpy.py

```diff
@@ -0,0 +1,50 @@
+"""
+============================================
+Getting the trajectory of Parker Solar Probe
+============================================
+
+This example shows how to plot the trajectory of Parker Solar Probe within the solar system using
+`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
+"""
+import datetime
+
+import matplotlib.pyplot as plt
+
+from astropy.time import Time
+
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates.ephemeris import get_horizons_coord
+
+##############################################################################
+# You can make use of `~datetime.datetime.now` to pass current date and time.
+# Now can use `get_horizons_coord() <https://docs.sunpy.org/en/stable/generated/api/sunpy.coordinates.get_horizons_coord.html>`__
+# to get the coordinates of Parker Solar Probe.
+
+today = datetime.datetime.now()
+obstime = Time(today)
+trajectory_coords = get_horizons_coord('Parker Solar Probe', {'start': '2021-10-11', 'stop': '2022-01-12', 'step': '180m'})
+
+##############################################################################
+# Now We are using `get_body_heliographic_stonyhurst()` to get the coordinates
```

Comment:
As I did in the previous example, you should be using the suggestion I made for this.

Reply:
Do you mean changing the value of step?

Reply:
No, I mean the text changes.

Reply:
Sorry, I didn't get that.